### ISSUE TITLE
rocWMMA support?

### ISSUE THREAD

**Issue by tedliosu** [ROCm/pytorch#1132]
### 🚀 The feature, motivation and pitch

Given how the competition has powerful pytorch extensions based on hardware-accelerated matrix FMA like tensorrt, and given how the impending gfx11 GPU releases have wmma support (similar to the competition's FMA-based tensorcores), I believe that it'd be really nice if pytorch-rocm is able to support GFX11's wmma instruction set (i.e. rocWMMA) via an extension similar to tensorrt or just natively via pytorch itself. Please refer to https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/issues/1899 for the relevant issue I created for the tensorflow software stack counterpart.

### Alternatives

Not really as I believe gfx 11 wmma support is the way to go moving forward :)

### Additional context

N/A

**Comment by taylding-amd** [comment]
Hi @tedliosu, thank you for sharing your suggestions. While PyTorch ROCm does not directly utilize the wMMA instructions, it does rely on mathematical libraries that take advantage of those instructions.
